### PR TITLE
[WOC] PR@2764: Update corner case canceling started issue

### PR DESCRIPTION
**Pull request by kuhnchris**
##### Description
Additional check for open == false but other things being true to show additional info message.

##### Checklist
- [ ] test

##### Affected core subsystem(s)
UI

##### Testing
Cannot test due to broken pyvips :-(

##### Refers/Fixes
Fixes: #2764

### CODE REVIEW

**Comment by mbeacom** [comment]
@kuhnchris I was hoping to include the restriction on the BE as well, but if you're opposed to adding it right now, I guess we can merge it and I'll try to tackle it at some point, time permitting.

**Comment by kuhnchris** [comment]
Your decision, that PR at least fixes the front-end side (for now).

**Comment by thelostone-mc** [comment]
@kuhnchris would be super neat if we could throw it into the BE if you are up for it ? ^_^

**Comment by mbeacom** [comment]
Anything you need from us here? @kuhnchris

**Comment by kuhnchris** [comment]
ah pardon me, didn't see the reply. want me to implement that check on backend side aswell (just that check) or want me to create a build-ticket to check this in general?

**Comment by thelostone-mc** [comment]
@kuhnchris would be ideal :P or do you just wanna resolve the conflicts and roll with this ?

**Comment by kuhnchris** [comment]
I do not mind either way. If the official statement is "fix be aswell" then I'll fix it, we just do not want to let this fall through the roof.

**Comment by kuhnchris** [comment]
@thelostone-mc you still know if this is still out there and causing issues? else I'd suggest we throw this away! Thanks!

File: app/assets/v2/js/pages/kill_bounty.js [hunk 1]
```diff
@@ -80,13 +80,18 @@ window.onload = function() {
...
           let errormsg = undefined;
 
-          if (bountyAmount == 0 || is_open == false) {
-            errormsg =
-                gettext('No active funded issue found at this address.  Are you sure this is an active funded issue?');
-          }
-          if (fromAddress != web3.eth.coinbase) {
-            errormsg =
-                gettext('Only the address that submitted this funded issue may kill the bounty.');
+          if (bountyAmount > 0 && !is_open) {
```
Comment (octavioamu) [inline app/assets/v2/js/pages/kill_bounty.js]: parseInt(0.09, 10)>0 will be false 
0.09 > 0 will be true (what we want)

**Pull request merged**